frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -59,3 +60,23 @@ export const getOCSVersion = (items: FirehoseResult): string => {
   );
   return _.get(operator, 'status.installedCSV');
 };
+
+export const getSCAvailablePVs = (sc: string) =>
+  k8sList(PersistentVolumeModel)
+    .catch(() => [])
+    .then((pvsData: K8sResourceKind[]) => {
+      return pvsData.filter(
+        (pv) =>
+          _.isEqual(_.get(pv, 'spec.storageClassName'), sc) &&
```

Comment:
https://github.com/openshift/console/pull/5016/files#diff-7879af0cb57fd55121c4d9598ff585ecR23

Reply:
Seems like you have pasted wrong link here @cloudbehl